client/components/LearnerViewDirective.js

```diff
@@ -278,6 +290,28 @@ tie.directive('learnerView', [function() {
         .tie-previous-instructions {
           opacity: 0.5;
         }
+        .tie-question-code {
+          background: rgb(242, 242, 242);
+          font-family: monospace;
+          padding: 10px;
+          border: 1px solid #ccc;
+          font-size: 13px;
+          white-space: pre-wrap;
+          white-space: -moz-pre-wrap;
+          white-space: -pre-wrap;
+          white-space: -o-pre-wrap;
+          word-wrap: break-word;  
+        }
+        .tie-question-example {
+          background: rgb(242, 242, 242);
+          padding: 10px;
+          border: 1px solid #ccc;
+          white-space: pre-wrap;
+          white-space: -moz-pre-wrap;
+          white-space: -pre-wrap;
+          white-space: -o-pre-wrap;
+          word-wrap: break-word;  
+        }
```

Comment:
Please order the css attributes in alphabetical order too, unless it impacts cascading.